apps/account-worker/src/worker.config.ts

```diff
@@ -3,6 +3,9 @@ import { registerAs } from '@nestjs/config';
 import Joi from 'joi';
 
 export interface IAccountWorkerConfig {
+  apiBodyJsonLimit: string;
+  workerApiPort: number;
```

Comment:
nit: all other services just call this `apiPort`, maybe rename for consistency (presumably in the other workers as well?)

```suggestion
  apiPort: number;
```